internal/core/items.rs

```diff
@@ -1164,22 +1152,7 @@ impl Opacity {
         if opacity == 1.0 {
             return false;
         }
-        let component_rc = self_rc.component();
-        let component_ref = vtable::VRc::borrow_pin(&component_rc);
-        let self_index = self_rc.index();
-        // TODO: use first_child() once it exists
-        let item_tree = component_ref.as_ref().get_item_tree();
-        let target_item_index = match item_tree.as_slice()[self_index] {
-            crate::item_tree::ItemTreeNode::Item { children_count, children_index, .. }
-                if children_count == 1 =>
-            {
-                children_index as usize
-            }
-            _ => return true, // Dynamic tree or multiple children -> need layer
-        };
-        let target_item = ItemRc::new(component_rc.clone(), target_item_index);
-        // any children? Then we need a layer
-        target_item.children_count() != Some(0)
+        self_rc.first_child().and_then(|target_item| target_item.first_child()).is_some()
```

Comment:
Iirc, we also said we would need to check that the opacity element only have one child

Reply:
I'm not sure I understand. I tried to make the code equivalent. I realize the above isn't, but I pushed what I think is a fix.

Reply:
If the Opacity item has two or more children (potentially overlapping)  we need to create a layer, which this code doesn't do as it only check the first children and doesn't check if it has a sibling.

If there is no children at all, no need to create a layer.

Reply:
I see now what you mean (I think ;-). Thanks for spotting.